docs/issue-labeling.md

```diff
@@ -0,0 +1,20 @@
+
+# Issue labeling guide
+
+
+## size/*
+size/* labels are for estimating size.
+It should be estimation how complicated and time consuming solving this is going to be.
+
+S - simple chage, just few lines (no more than 1 day of work)
+M - a little bit more change but still fairly strait forward
+L - a bit more complicated to solve, maybe requiring small refactoring of existing code
+XL - complex change, new big feature, requiring big refactoring
```

Comment:
XL - complex change, new big feature, requiring big refactoring, perhaps is an epic and should be broken into smaller tasks.